### PR TITLE
Booking request

### PR DESCRIPTION
**Pull request by jdm79**
creates a booking request table with date. users can now request bookings. however, there is now an issue with the css on the description box where it's not going to the next line if a lot of text is inputted.

### CODE REVIEW

File: app/controllers/bookings.rb [hunk 1]
```diff
@@ -1,10 +1,18 @@
 
 class Bookings < Sinatra::Base
 
-  get '/bookings' do
+  get '/bookings/:id' do
+    @property = Property.find_by(id: params[:id].to_i)
+    p @property
```
Comment (SecretSurfSpot) [inline app/controllers/bookings.rb]: probably should remove p statements now

File: app/controllers/bookings.rb [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
+    erb :booking, :layout => :layout_logged_in
   end
 
-  post '/bookings' do
+  post '/bookings/:id' do
+    booking = { account_id: session[:user_id], property_id: params[:id], date: params[:bookingdate], confirmed: false }
+    Booking.create(booking)
+
+    p params
```
Comment (SecretSurfSpot) [inline app/controllers/bookings.rb]: remove p commands

Comment (Mallig) [reply app/controllers/bookings.rb]: thanks :)

**Review by SecretSurfSpot — COMMENTED** [review]
Other than the 2 p commands - looks good to me.

**Pull request closed without merging**